.github/workflows/tests.yaml

```diff
@@ -31,9 +31,8 @@ jobs:
         postgis: [false]
         dev: [false]
         env:
-          - ci/envs/36-minimal.yaml
+          - ci/envs/37-minimal.yaml
           - ci/envs/38-no-optional-deps.yaml
-          - ci/envs/36-pd025.yaml
```

Comment:
I would maybe keep this, but move it eg to python 3.7, and eg pin pandas to 1.0 (some intermediate release)

Reply:
Okay, it is back.